src/main.tsx

```diff
@@ -1,16 +1,14 @@
 import ReactDOM from 'react-dom/client'
 import App from './App'
 import { ChakraProvider, ColorModeScript } from '@chakra-ui/react'
-import { theme } from './theme/index'
+import { theme } from './theme'
 import { ApplicationProvider } from './contexts/applicationContext'
 import { SettingsProvider } from './contexts/settingsContext'
 import '@fontsource/montserrat/800.css'
-import { attachConsole } from 'tauri-plugin-log';
-
-
-// with LogTarget::Webview enabled this function will print logs to the browser console
-attachConsole();
+import { attachConsole } from 'tauri-plugin-log'
 
+// with LogTarget::Webview enabled, this function will print logs to the browser console
```

Comment:
linitng was complaining so tried to fix

Reply:
If linting is complaining, then I would rather disable the linting error for this line than adding a redundant bit of code that happens to stop it